Puppetfile

```diff
@@ -0,0 +1,3 @@
+forge "http://forge.puppetlabs.com"
```

Comment:
Why do you need this? I think using `Puppetfile` is depreciated ...

Reply:
I remember 4 years ago librarian was state of the art for puppet, is not the case anymore? I took a look to kitchen-puppet and still see librarian there, is there any other way to get the dependencies?

Reply:
Well, I just know `Puppetfile` out of metadata perspective, and there it's not used any more ... but it seems to be used for automatic environment management / deployment, so it's still valid  :-)

Reply:
great :-)